File: frontend/templates/kotlin/Product.kt
import java.net.URL

fun main() {
  try {
    val response = URL("https://real-time-backend-preview.vercel.app/api/product").readText()
    println("Product: $response")
  } catch (e: Exception) {
    println("Error: ${e.message}")
  }
}

File: frontend/templates/csharp/Product.cs
using System;
using System.Net.Http;
using System.Threading.Tasks;

class Program {
  static async Task Main() {
    using var client = new HttpClient();
    try {
      var response = await client.GetStringAsync("https://real-time-backend-preview.vercel.app/api/product");
      Console.WriteLine("Product: " + response);
    } catch (Exception e) {
      Console.WriteLine("Error: " + e.Message);
    }
  }
}